src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mlp_seq.cpp

```diff
@@ -0,0 +1,124 @@
+                                            ::testing::ValuesIn(precision_f32(1)),
+                                            ::testing::Values(ov::element::f32),
+                                            ::testing::ValuesIn({static_cast<size_t>(1), MLPQuantized::default_thread_count}),
+                                            ::testing::Values(1),  // Subgraph
+                                            ::testing::Values(1),  // MLPQuantized
+                                            ::testing::Values(ov::test::utils::DEVICE_CPU),
+                                            ::testing::Values(CPUTestUtils::empty_plugin_config),
+                                            ::testing::ValuesIn(numHiddenLayers()),
+                                            ::testing::ValuesIn(hiddenMatmulSizes())),
+                         MLPQuantized::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MLP_SEQ_Quantized_2D_i8,
+                         MLPQuantized,
+                         ::testing::Combine(::testing::ValuesIn(inputShape_2D()),
+                                            ::testing::ValuesIn(precision_i8(1)),
+                                            ::testing::Values(ov::element::f32),
+                                            ::testing::ValuesIn({static_cast<size_t>(1), MLPQuantized::default_thread_count}),
+                                            ::testing::Values(1),  // Subgraph
+                                            ::testing::Values(1),  // MLPQuantized
+                                            ::testing::Values(ov::test::utils::DEVICE_CPU),
+                                            ::testing::Values(CPUTestUtils::empty_plugin_config),
+                                            ::testing::ValuesIn(numHiddenLayers()),
+                                            ::testing::ValuesIn(hiddenMatmulSizes())),
+                         MLPQuantized::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MLP_SEQ_2D_f32_EnforceBF16,
+                         MLP,
+                         ::testing::Combine(::testing::ValuesIn(inputShape_2D()),
+                                            ::testing::ValuesIn(precision_f32(1)),
+                                            ::testing::Values(ov::element::bf16),
+                                            ::testing::ValuesIn({static_cast<size_t>(1), MLP::default_thread_count}),
+                                            ::testing::Values(1),  // Subgraph
+                                            ::testing::Values(1),  // MLP
+                                            ::testing::Values(ov::test::utils::DEVICE_CPU),
+                                            ::testing::Values(CPUTestUtils::empty_plugin_config),
+                                            ::testing::ValuesIn(numHiddenLayers()),
+                                            ::testing::ValuesIn(hiddenMatmulSizes())),
+                         MLP::getTestCaseName);
```

Comment:
Can we combine it with `smoke_Snippets_MLP_SEQ_2D_f32`?

Reply:
Yes, done